modules/packages/LinearAlgebra.chpl

```diff
@@ -1976,6 +2017,47 @@ module Sparse {
     }
     return A;
   }
+  
+ /* Compute the approximate solution to ``A * x = b`` using the Jacobi method.
+    Iteration will stop when ``maxiter`` is reached or error is smaller than
+    ``tol``, whichever comes first. Return the number of iterations performed.
+    
+    .. note::
+      ``X`` is passed as a reference, meaning the initial solution guess can be
+      stored in ``X`` before calling the procedure, and the approximate solution 
+      will be stored in the same array.
+      
+    */
```

Comment:
Let's follow the same thing as the other PRs and "no doc" this one, And mention that the jacobi supports CSR arrays in the documentation above.